es/cheatsheet-unsupervised-learning.md

```diff
@@ -1,340 +1,342 @@
 **1. Unsupervised Learning cheatsheet**
 
-&#10230;
+&#10230; Aprendizaje no Supervisado
 
 <br>
 
 **2. Introduction to Unsupervised Learning**
 
-&#10230;
+&#10230; Introducción al Aprendizaje no Supervisado
 
 <br>
 
 **3. Motivation ― The goal of unsupervised learning is to find hidden patterns in unlabeled data {x(1),...,x(m)}.**
 
-&#10230;
+&#10230; Motivación - El objetivo del aprendizaje no supervisado es encontrar patrones ocultos en datos no etiquetados {x(1),...,x(m)}.
 
 <br>
 
 **4. Jensen's inequality ― Let f be a convex function and X a random variable. We have the following inequality:**
 
-&#10230;
+&#10230; Desigualdad de Jensen - Sea f una función convexa y X una variable aleatoria. Tenemos la siguiente desigualdad:
 
 <br>
 
 **5. Clustering**
 
-&#10230;
+&#10230; Clusterización (Agrupamiento)
 
 <br>
 
 **6. Expectation-Maximization**
 
-&#10230;
+&#10230; Expectativa-Maximización
 
 <br>
 
 **7. Latent variables ― Latent variables are hidden/unobserved variables that make estimation problems difficult, and are often denoted z. Here are the most common settings where there are latent variables:**
 
-&#10230;
+&#10230; Variables Latentes - Las variables latentes son variables ocultas / no observadas que dificultan los problemas de estimación y a menudo son denotadas como z. Estos son los ajustes más comunes más comunes en los que hay variables latentes:
 
 <br>
 
 **8. [Setting, Latent variable z, Comments]**
 
-&#10230;
+&#10230; [Ajustes, Variable latente z, Comentarios]
 
 <br>
 
 **9. [Mixture of k Gaussians, Factor analysis]**
 
-&#10230;
+&#10230; [Mezcla de k Gaussianos, Análisis factorial]
 
 <br>
 
 **10. Algorithm ― The Expectation-Maximization (EM) algorithm gives an efficient method at estimating the parameter θ through maximum likelihood estimation by repeatedly constructing a lower-bound on the likelihood (E-step) and optimizing that lower bound (M-step) as follows:**
 
-&#10230;
+&#10230; Algoritmo - El algoritmo Expectativa-Maximización (EM) proporciona un método eficiente para estimar el parámetro θ a través de la estimación de máxima probabilidad construyendo repetidamente un límite inferior en la probabilidad (E-step) y optimizando ese límite inferior (M-step) de la siguiente manera:
```

Comment:
The technical term for "maximum likelihood estimation" is "_[estimación por máxima similitud](https://es.wikipedia.org/wiki/M%C3%A1xima_verosimilitud)_".

Reply:
likelihood is a tricky word, most dictionaries translate "likelihood" as "probabilidad" although the word  likelihood differs from probability . I think we should stick with "verosímil".